traefik/templates/deployment.yaml

```diff
@@ -119,8 +119,11 @@ spec:
           {{- end }}
           {{- if .Values.providers.kubernetesIngress.enabled }}
           - "--providers.kubernetesingress"
-          {{- end }}          
-          {{- if and .Values.rbac.enabled .Values.rbac.namespaced}}
+          {{- if .Values.providers.kubernetesIngress.publishedService.enabled }}
```

Comment:
I suspect that if the user disables the `kubernetesIngress` provider, but leaves the `publishService` enabled it will result in a misconfigured traefik that will not start.

should we consider adding a condition check for `kubernetesIngress.enabled` here as well?

Reply:
Hi @notsureifkevin,
Thank you for the review. I tested this case and nothing happened because all `publishedService` is inside the if of `kubernetesIngress enabled`. But I discover a misconfiguration if `service.enabled` is false and I added an extra check.

Is it resolved for you?